.travis.yml

```diff
@@ -2,9 +2,6 @@ language: python
 
 matrix:
   include:
-  - python: "3.7"
```

Comment:
no more 3.7? I'm totally fine with this, but this seems like a decision.

Reply:
We are no longer compatible with Python 3.7 -- the `zipfile.Path()` stuff was only added to the standard Python library in 3.8.